libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -683,6 +683,9 @@ class AP_AHRS {
     // get access to an EKFGSF_yaw estimator
     const EKFGSF_yaw *get_yaw_estimator(void) const;
 
+    void set_data_send_disable(bool disable);
```

Comment:
The name isn't apparent what this does. Can you add a comment? 

Does this disable the EARHS from sending data back to the autopilot?

Reply:
I added the comment

Reply:
Done